tools/docker/Dockerfile

```
@@ -5,6 +5,7 @@ ARG STATIC_ROOT=/var/lib/eda/static
 
 # python3.11 and python3.12 supported
 ARG PYTHON_BIN="python3.11"
+ARG ANSIBLE_CORE_VER=${ANSIBLE_CORE_VER:-2.16}
```

Build args are overridable by default. 
```suggestion
ARG ANSIBLE_CORE_VER=2.16
```

---

The idea was to allow it to be set via env var, like we do with USER_ID